code/modules/clothing/rogueclothes/leather/pants.dm

```diff
@@ -0,0 +1,31 @@
+
+/obj/item/clothing/under/roguetown/trou/leather
+	name = "leather trousers"
+	desc = "Trousers made of fine leather."
+	icon_state = "leatherpants"
+	max_integrity = 130
+	armor = list("blunt" = 50, "slash" = 25, "stab" = 40, "bullet" = 10, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	prevent_crits = list(BCLASS_CUT, BCLASS_STAB)
+
+/obj/item/clothing/under/roguetown/trou/leather/advanced
+	name = "hardened leather chausses"
+	desc = "Sturdy, durable, flexible. The finest leather to protect your nether regions."
+	icon_state = "leatherpants_masterwork" //no advanced sprite
```

Comment:
So masterowkr and hardened keep the same icon? wouldn't it be better to keep Cynthias on-mob sprites for hardened leather just so theres a difference?